EventListener/AttachRefreshTokenOnSuccessListener.php

```diff
@@ -107,9 +115,27 @@ public function attachRefreshToken(AuthenticationSuccessEvent $event): void
             $refreshToken = $this->refreshTokenGenerator->createForUserWithTtl($user, $this->ttl);
 
             $this->refreshTokenManager->save($refreshToken);
-            $data[$this->tokenParameterName] = $refreshToken->getRefreshToken();
+            $refreshTokenString = $refreshToken->getRefreshToken();
         }
 
-        $event->setData($data);
+        if ($this->cookie) {
```

Comment:
If the suggestion about adding `canBeEnabled()` in the config is applied, then this should be changed.  Otherwise, as pointed out above, right now this enables itself based on whether the config array is empty or not.